org-node/src/lib.rs

```diff
@@ -41,6 +44,7 @@ pub struct Options {
     pub cache: PathBuf,
     pub identity: PathBuf,
     pub bootstrap: Vec<(PeerId, net::SocketAddr)>,
+    pub rpc_url: String,
     pub listen: net::SocketAddr,
     pub subgraph: String,
     pub poll_interval: time::Duration,
```

Comment:
We can here also remove the `poll_interval` option.
Also on line 36 the `DEFAULT_POLL_INTERVAL`